test/core/rebalancingSetToken.spec.ts

```diff
@@ -41,22 +52,40 @@ contract('RebalancingSetToken', accounts => {
 
   let rebalancingSetToken: RebalancingSetTokenContract;
   let components: StandardTokenMockContract[] = [];
+
+  let core: CoreMockContract;
```

Comment:
wonder if we should rename this all to `coreMock` to be clear

Reply:
I don't see a reason to. We're only using CoreMock in this test suite...but it's still what we're using as Core.

Reply:
+1, let's use coreMock here when it's coreMock